server/src/main/java/org/spine3/server/event/enrich/EventEnricher.java

```diff
@@ -55,16 +55,18 @@
 import static org.spine3.protobuf.Messages.toMessageClass;
 
 /**
- * {@code Enricher} extends information of an event basing on its type and content.
+ * {@code EventEnricher} extends information of an event basing on its type and content.
```

Comment:
I wanted to have latest `EventEnricher` in  the code from Projects. That's why this PR is based on earlier works.